hspec-discover/README.md

```diff
@@ -0,0 +1,17 @@
+# hspec-discover
+
+Automatically discover and run Hspec tests.
+
+It is a useful convention to have one spec file for each source file. That way it is straightforward to find the corresponding spec for a given piece of code. But it requires error prone, and neither challenging nor interesting boiler plate code. So it should be automated. Hspec provides a solution for that. It makes creative use of GHC's support for custom preprocessors.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+*TODO: Add instructions for usage*
```

Comment:
Would you mind adding a short usage description for how this would be use in the context of Habitat?

Reply:
Done.